src/com/lftechnology/java/training/sanish/numbergame/Guess.java

```diff
@@ -0,0 +1,133 @@
+package com.lftechnology.java.training.sanish.numbergame;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Guess number program in between 1 - 20
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class Guess {
```

Comment:
Guess Number name would be more specific since the game is about only guessing numbers